napari_allencell_annotator/view/annotator_view.py

```diff
@@ -213,12 +218,18 @@ def render_values(self, vals: List):
         vals:List
             the values for the annotations.
         """
-        for (widget, val) in zip(self.annotation_item_widgets, vals):
```

Comment:
nice work on this function pretty clever

Reply:
thanks!